azureadps-2.0/AzureAD/Get-AzureADDirectoryRole.md

```diff
@@ -44,7 +44,7 @@ PS C:\>Get-AzureADDirectoryRole
 
 ObjectId                             DisplayName                        Description
 --------                             -----------                        -----------
-019ea7a2-1613-47c9-81cb-20ba35b1ae48 Global Administrator               Can manage all aspects of Azure AD and Microsoft services that use Azure AD identities.
+aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb Global Administrator               Can manage all aspects of Azure AD and Microsoft services that use Azure AD identities.
```

Comment:
Should the role IDs be updated? Could be because of the ObjectId value. Then all the ObjectIds in the output need to be updated.

Reply:
This is an allowed value, and should be reverted